lib/strava-ruby-client.rb

```diff
@@ -5,6 +5,7 @@
 require 'hashie'
 require 'time'
 
+require 'active_support' if ENV['RACK_ENV'] == 'test'
```

Comment:
Previous versions of ActiveSupport let you pick and choose features and included common `active_support` in each file. I believe that was removed. Make this `require 'active_support'` always, remove the ENV check. We definitely don't want gem code to behave differently in dev/test/prod.